docs/hello_nf-core/04_make_module.md

```diff
@@ -133,44 +149,77 @@ Test the workflow to ensure metadata flows through correctly:
 nextflow run . --outdir core-hello-results -profile test,docker --validate_params false
 ```
 
-The pipeline should run successfully with metadata now flowing from `CAT_CAT` through `cowpy`.
+The pipeline should run successfully with metadata now flowing from `CAT_CAT` through `cowpy`:
+
+```console title="Output (excerpt)"
+executor >  local (8)
+[b2/4cf633] CORE_HELLO:HELLO:sayHello (2)       [100%] 3 of 3 ✔
+[ed/ef4d69] CORE_HELLO:HELLO:convertToUpper (3) [100%] 3 of 3 ✔
+[2d/32c93e] CORE_HELLO:HELLO:CAT_CAT (test)     [100%] 1 of 1 ✔
+[da/6f3246] CORE_HELLO:HELLO:cowpy              [100%] 1 of 1 ✔
+-[core/hello] Pipeline completed successfully-
+```
 
 ### 1.2. Simplify the interface with ext.args
 
 Now let's address another nf-core pattern: simplifying module interfaces by using `ext.args` for optional command-line arguments.
 
-Currently, our `cowpy` module requires the `character` parameter to be passed as a separate input. While this works, nf-core modules follow a convention of keeping interfaces minimal - only essential inputs (metadata and files) should be declared. Optional tool arguments are instead passed via configuration.
+Currently, our `cowpy` module requires the `character` parameter to be passed as a separate input. While this works, nf-core modules use a different approach for **tool configuration arguments**: instead of adding input parameters for every tool option, they use `ext.args` to pass these via configuration. This keeps the module interface focused on essential data (files, metadata, and any mandatory per-sample parameters), while tool configuration options are handled through `ext.args`.
```

Comment:
I find the phrasing 'as a separate input' quite confusing here. Separate from what? To be clear I know what it's trying to say but I think for the novice user this is a headscratcher.

Reply:
Fair, can be reworded